code/game/gamemodes/objective.dm

```diff
@@ -306,6 +306,29 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	name = "protect nonhuman"
 	human_check = FALSE
 
+/datum/objective/jailbreak
+	name = "jailbreak"
+	var/target_role_type=FALSE
+	martyr_compatible = 1 //why not?
```

Comment:
Move this line up one and set it to `TRUE`

Reply:
which bit?

Reply:
He means the Martyr_Comparible var, and it reads better to have new vars for any subtype below the parent vars.

Reply:
Oh thanks

Reply:
fixed